src/Dapr.Extensions.Configuration/DaprSecretStoreConfigurationSource.cs

```diff
@@ -18,7 +18,7 @@ public class DaprSecretStoreConfigurationSource : IConfigurationSource
         /// <summary>
         /// Gets or sets the store name.
         /// </summary>
-        public string Store { get; set; }
+        public string? Store { get; set; }
```

Comment:
Would prefer that this not be made nullable

Reply:
Used an empty string as the default value here. This will raise an `ArgumentException` in the constructor of `DaprSecretStoreConfigurationProvider`.